src/data/dropdownDataGroupedDescriptions.tsx

```diff
@@ -0,0 +1,114 @@
+import {Folder, Lock} from '~/icons';
+
+export const dropdownGroupedDataDescriptions = [
+    {
+        groupLabel: 'test for item list with image',
```

Comment:
Rename the group label because we don't use image anymore. Propostion : `Group with description`